awx/api/serializers.py

```diff
@@ -1757,6 +1758,109 @@ class Meta:
         fields = ()
 
 
+class BulkHostSerializer(serializers.Serializer):
```

Comment:
I want to ask the question - could this inherit from `HostSerializer`?

Even so, I expect this needs some extra logic beyond the base serializer, to error if the "inventory" field is provided and is different from the top-level inventory value.

Reply:
I got it to work inheriting from HostSerializer so we can share the validation